app/server.go

```diff
@@ -151,6 +151,11 @@ func NewServer(options ...Option) (*Server, error) {
 		s.configStore = configStore
 	}
 
+	// Enable plaintext logger output if this is a "dev" build, but don't override existing settings.
+	if model.BuildNumber == "dev" && s.configStore.FromDefaults() {
+		s.UpdateConfig(func(cfg *model.Config) { *cfg.LogSettings.ConsoleJson = false })
```

Comment:
Separately, while this does indeed get us non-structured log output by default, there's an `mlog/human` package that @zeroimpl contributed to make those logs even more human readable -- effectively leveraging logrus to get all the pretty rendering and such.

Ideally, we'd find a way to enable that for developers by default. I'm hesitant to introduce a new bit in the `LogSettings` configuration structure for something that we don't want to expose in a production setting right now. Crazy idea: what if we piped `$(GO) run ...` to `$(GO) run .. logs ...`, leaving the original JSON structured output but routing it through a second invocation to post-process those logs, just like the `mattermost logs` command does today?

Reply:
Hi Jesse!
I think this makes more sense, I just pushed and ammended. I think this one line change does everything you want.

Thanks for the suggestion, really clever solution.

Reply:
@lieut-data I just read https://developers.mattermost.com/blog/submitting-great-prs/ , apologies about ammending and force pushing the new change. :)